assets/js/meetup-api/exec.js

```diff
@@ -0,0 +1,33 @@
+function ParsedEvents(rawEvents) {
```

Comment:
tu się nie zgodzę, bo to nie jest metoda tlyko de facto deklaracja "klasy", tak to się robi w js :P to samo jest z MeetupApi() w client.js